dev-support/docker/vcpkg/vcpkg.json

```diff
@@ -10,7 +10,7 @@
   "overrides": [
     {
       "name": "protobuf",
-      "version": "3.21.12"
+      "version": "3.25.5"
```

Comment:
I suppose this works, but I don't have Windows env to verify this.

cc @GauthamBanasandra

Reply:
@GauthamBanasandra Now it passes on all Linux containers. Due to my limited CPP experience, my change might be dirty ...

Reply:
@GauthamBanasandra do you remember why we should install `libprotobuf-dev` and `libprotoc-dev` through APT? seems we can remove it and always use the manually installed protobuf instead?